apps/libsignpost/signpost_api.c

```diff
@@ -856,13 +853,18 @@ static bool energy_query_ready;
 static int  energy_query_result;
 static signbus_app_callback_t* energy_cb = NULL;
 static signpost_energy_information_t* energy_cb_data = NULL;
+static bool energy_report_received;
 
 static void energy_query_sync_callback(int result) {
     SIGNBUS_DEBUG("result %d\n", result);
     energy_query_ready = true;
     energy_query_result = result;
 }
 
+static void signpost_energy_report_callback(__attribute__ ((unused)) int result) {
```

Comment:
Is it really safe to not use this result? It's quite possibly than an async error bubbles up